mlir/include/Gradient/IR/GradientOps.td

```diff
@@ -92,4 +92,33 @@ def AdjointOp : Gradient_Op<"adjoint", [AttrSizedOperandSegments]> {
     }];
 }
 
+def BackpropOp : Gradient_Op<"backprop", [AttrSizedOperandSegments]> {
+    let summary = "Perform classic automatic differentiation using Enzyme AD.";
+
+    let arguments = (ins
+        FlatSymbolRefAttr:$callee,
+        Index:$gradSize,
+        Variadic<AnyType>:$args,
+        Variadic<AnyMemRef>:$data_in,
+        OptionalAttr<AnyIntElementsAttr>:$diffArgIndices
```

Comment:
Ok for sure!

Reply:
I don't know if it is good practice for an attribute but I added it to the tablegen in assembly format with `argnums`

Reply:
Oh generally there is no need for that unless you want to provide some custom syntax for the attribute, otherwise all attributes are just captured via the `attr-dict` command in the custom assembly.

Reply:
Ok thanks I will change it!